### PR TITLE
code with a mistake

### PR DESCRIPTION
**Pull request by basd2844**

### CODE REVIEW

File: Spikey.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿public class Spikey
+{
+    public static void main(String[] args)
+    {
+        System.out.println("\\/");
```
Comment (arnavm7) [inline Spikey.java]: this line of code needs 2 spaces before and after the slashes

File: Spikey.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        System.out.println(“\\\\//");
```
Comment (arnavm7) [inline Spikey.java]: this line of code needs 1 space before and after the slashes

File: Spikey.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        System.out.println("\\\\\\///");
+        System.out.println("///\\\\\\");
+        System.out.println(“//\\\\");
```
Comment (arnavm7) [inline Spikey.java]: this line of code needs a space before and after the slashes

File: Spikey.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        System.out.println("/\\");
```
Comment (arnavm7) [inline Spikey.java]: this line of code needs 2 spaces before and after the slashes

**Comment by twang** [comment]
Thanks @basd2844 and @arnavm7!

**Pull request closed without merging**